frontend/website/src/InvestorsSection.re

```diff
@@ -0,0 +1,102 @@
+module Investor = {
+  let component = ReasonReact.statelessComponent("BuiltBy.Investor");
+  let make = (~name, ~suffix, _) => {
+    let firstName = Js.String.split(" ", name)[0];
+    let suffixStr =
+      switch (suffix) {
+      | `Png => "png"
+      | `Jpg => "jpg"
+      };
+    let imageSrc =
+      "/static/img/investors/"
+      ++ String.lowercase(firstName)
+      ++ "."
+      ++ suffixStr;
+    {
+      ...component,
+      render: _self =>
+        <div
+          className=Css.(
+            style([
+              display(`flex),
+              flexDirection(`row),
+              alignItems(`center),
+            ])
+          )>
+          <img
+            src=imageSrc
+            className=Css.(
+              style([
+                width(`rem(2.5)),
+                height(`rem(2.5)),
+                borderRadius(`percent(100.)),
+                marginRight(`em(1.)),
+                boxShadow(
+                  ~x=`zero,
+                  ~y=`zero,
+                  ~blur=`px(4),
+                  ~spread=`zero,
+                  Style.Colors.greenShadow,
+                ),
+              ])
+            )
+          />
+          <h4
+            className=Css.(
+              merge([Style.Body.basic, style([color(Style.Colors.slate)])])
+            )>
+            {ReasonReact.string(name)}
+          </h4>
+        </div>,
+    };
+  };
+};
+
+let component = ReasonReact.statelessComponent("BuiltBy");
+let make = _children => {
+  ...component,
+  render: _self =>
+    <div>
+      <h3 className=Style.H3.wide> {ReasonReact.string("Investors")} </h3>
+      <div
+        className=Css.(
+          style([
+            maxWidth(`rem(64.)),
+            display(`grid),
+            gridGap(`rem(1.)),
+            marginTop(`rem(3.)),
+            marginLeft(`auto),
+            marginRight(`auto),
+            justifyContent(`center),
+            gridTemplateColumns([`fr(1.), `fr(1.)]),
+            media(
+              Style.MediaQuery.notMobile,
+              [
+                // bs-css doesn't allow fr in minmax: https://github.com/SentiaAnalytics/bs-css/pull/124
+                `declaration((
```

Comment:
you can use `unsafe("foo", "whatever")` also (this is what I've been using in grid)